tests/marketplace-tx.test.ts

```diff
@@ -1,58 +1,63 @@
-
-import { DirectSecp256k1HdWallet, SigningStargateClient, StargateClient, GasPrice, AccountData, PageRequest } from '../src/index';
+import { DirectSecp256k1HdWallet, SigningStargateClient, StargateClient, GasPrice, AccountData } from '../src/index';
 import { coin } from '@cosmjs/amino';
 import Long from 'long';
-import { Royalty } from '../src/stargate/modules/marketplace/proto-types/royalty';
 
 describe('marketplace', () => {
 
-  const mnemonic1 = 'rebel wet poet torch carpet gaze axis ribbon approve depend inflict menu'
-  const mnemonic2 = 'noise memory wonder bottom kiwi device ranch scrub gasp also illegal grunt'
+  const mnemonic4 = 'debate result toward brick dutch unusual caught plunge honey ramp want update'
+  const mnemonic5 = 'tape soul absorb cabin luxury practice rally clerk love kiss throw avoid'
 
-  const gasPrice = GasPrice.fromString('5000000000000acudos');
+  const gasPrice = GasPrice.fromString('6000000000000acudos');
 
-  const rpc = 'http://0.0.0.0:26657/';
-  let faucetWallet: DirectSecp256k1HdWallet;
-  let faucetAccount: AccountData;
-  let faucetAddress: string;
-  let faucet: SigningStargateClient;
+  const rpc = 'http://localhost:26657';
+  let mainWallet: DirectSecp256k1HdWallet;
+  let mainAccount: AccountData;
+  let mainAddress: string;
+  let mainSigningClient: SigningStargateClient;
   let queryClient: StargateClient;
   let aliceWallet: DirectSecp256k1HdWallet;
   let aliceAccount: AccountData;
   let aliceAddress: string;
   let alice: SigningStargateClient;
 
-
   jest.setTimeout(40000);
-  // deploying alpha contract once before test cases
+
   beforeAll(async () => {
-    faucetWallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic1);
-    faucetAccount = (await faucetWallet.getAccounts())[0];
-    faucetAddress = faucetAccount.address;
-    faucet = await SigningStargateClient.connectWithSigner(rpc, faucetWallet);
+    mainWallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic4);
```

Comment:
can we extract this into a constant utility function, something like "setup env" or "setp tests"

Reply:
sure